packages/amplify-frontend-android/lib/frontend-config-creator.js

```diff
@@ -1,6 +1,48 @@
 const constants = require('./constants');
 const path = require('path');
 const fs = require('fs-extra');
+const graphQLConfig = require('graphql-config');
+
+const FILE_EXTENSION_MAP = {
+  javascript: 'js',
+  graphql: 'graphql',
+  flow: 'js',
+  typescript: 'ts',
+  angular: 'graphql',
+};
+
+const fileNames = ['queries', 'mutations', 'subscriptions'];
+
+function deleteAmplifyConfig(context) {
+  const { amplify } = context;
+  const projectPath = context.exeInfo ? context.exeInfo.localEnvInfo.projectPath : amplify.getEnvInfo().projectPath;
+  const projectConfig = context.exeInfo ? context.exeInfo.projectConfig[constants.Label] : amplify.getProjectConfig()[constants.Label];
+  const frontendConfig = projectConfig.config;
+  const srcDirPath = path.join(projectPath, frontendConfig.ResDir, 'raw');
+  const targetFilePath = path.join(srcDirPath, constants.amplifyConfigFilename);
+  if (fs.existsSync(targetFilePath)) {
+    fs.removeSync(targetFilePath);
+  }
+
+  const awsConfigFile = path.join(srcDirPath, constants.awsConfigFilename);
+  if (fs.existsSync(awsConfigFile)) {
+    fs.removeSync(awsConfigFile);
+  }
+
+  if (!fs.existsSync(path.join(projectPath, '.graphqlconfig.yml'))) return;
+
+  const gqlConfig = graphQLConfig.getGraphQLConfig(projectPath);
+  if (gqlConfig && gqlConfig.config) {
+    const { projects } = gqlConfig.config;
+    Object.keys(projects).forEach(project => {
+      const { codeGenTarget, docsFilePath } = projects[project].extensions.amplify;
+      fileNames.forEach(filename => {
+        const file = path.join(projectPath, docsFilePath, `${filename}.${FILE_EXTENSION_MAP[codeGenTarget] || 'graphql'}`);
```

Comment:
@yuth Can you please verify this?

Reply:
const file = path.join(projectPath, docsFilePath, `${filename}.${FILE_EXTENSION_MAP[codeGenTarget] || 'graphql'}`); 

Not sure if this logic is correct..can't we have both the graphql directory and a swift or TS file?

Reply:
The codeGenTarget doesn't have any files so I default it to 'graphql'